drned-skeleton/load-default-config.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import print_function
```

Comment:
The module name is not good - it does not matter much since it is used as a standalone Python script, but it is adviced to use "snake case" for module names: using hyphens in module names prevent makes function calls like `load-default-config.load_default_config(...)` impossible.